packages/translations/src/exports/all.ts

```diff
@@ -44,6 +46,8 @@ export const translations = {
   ar,
   az,
   bg,
+  bnBD,
```

Comment:
These should likely be `'bn-BD': bnBD`, like how `zh-TW` is set at the very bottom